_posts/2015-03-10-updated-site.markdown

```diff
@@ -0,0 +1,47 @@
+
+# What to expect with the site for 0.8
+
+0.8 will be a new baseline for feedback - it will be hot out of the oven, but able to stand on its own. Consider it an "alpha" release - rough around the edges but generally whole, with the express purpose of getting feedback to roll into 1.0.
+
+With this in mind, we're working to accompany the release with a baseline of documentation to help early-adopters get started - expect the documentation to grow significantly as the features bake. The site itself for the time being will continue to be built based on 0.5 - we want to get 0.8 out for wide feedback as soon as possible, so having some documentation ready and starting to create 0.8-compatible elements has taken a priority over updating the site's library itself.
+
+We're also setting up the ability to version the documentation on the site - you'll notice that [https://www.polymer-project.org/](https://www.polymer-project.org/) currently takes you to https://www.polymer-project.org/0.5/ (the latest stable version). 0.8 will be transitional for the site as it is for the library and we'll be growing into this new versioning feature, but expect more and more firm structure like this to emerge.
+
+## What should you do now?
+
+### Update hotlinked URL's
+
+Many have found it helpful to be able to hotlink to the Polymer library hosted on the Polymer site, to lower develop-time overhead and to be able to easily create jsbin-like examples. With the new site versioning model, URL's will be of the form:
+
+`https://www.polymer-project.org/{version}/...`
+
+We certainly don't recommend relying on hotlinking for anything production. This new versioning should help hotlinking for development be more stable going forward, and help clarify any jsbin examples with an explicit version number.
+
+With regards to elements - we're working on major updates to the elements built by the Polymer team (you can see some progress in Github branches), including a better way of showcasing the element sets than we current have on the site. We'll keep the 0.5 elements the same as they are currently on the site. For 0.8, we plan on keeping compatible elements on Github for now as they're being built, and are working on new and improved ways to demo elements and generate and view their documentation.
+
+### Make sure to pin your `bower.json` versions
+
+The 0.8 release will be a significant breaking change, so make sure you pin the version of the library you request in your `bower.json` file, so that `bower update` won't break your project. Also make sure to explicitly pin element versions, as we may release 0.8-compatible versions of elements in their current repository when they're ready.
+
+For example, to pin to `0.5.5`:
+
+
+    "dependencies": {
+      "polymer": "polymer/polymer#0.5.5",
```

Comment:
Might want to suggest Polymer/polymer#^0.5.5 to allow for future patches to the 0.5 release, if required.